src/usr/local/www/widgets/widgets/system_information.widget.php

```diff
@@ -334,6 +335,7 @@ function get_sysinfo_item_html($itemkey) {
 				break;
 
 			case 'ALL_HIDDEN':
+				$title_content = gettext('Empty');
```

Comment:
Helps to see something in the "item title" for the empty item. Can go in master.